diff -ur ruby-password-0.5.3-o/extconf.rb ruby-password-0.5.3/extconf.rb
--- ruby-password-0.5.3-o/extconf.rb	2006-03-02 10:35:06.000000000 -0700
+++ ruby-password-0.5.3/extconf.rb	2006-09-27 14:20:57.000000000 -0600
@@ -12,6 +12,7 @@
   /usr/lib/cracklib_dict.pwd
   /opt/lib/cracklib_dict.pwd
   /var/cache/cracklib/cracklib_dict.pwd
+	/usr/share/dict/cracklib_dict.pwd
 )
 
 if dict = with_config('crack-dict')
